projects/discussion-ui/src/lib/services/config.service.ts

```diff
@@ -70,4 +70,8 @@ export class ConfigService implements OnInit {
   public getRouterSlug() {
     return this._config.routerSlug ? this._config.routerSlug : '';
   }
+
+  public getHeaderOption() {
```

Comment:
Why this method is added?

Reply:
This method has been added to remove the back and cancel button from igot. And these are main configurable.